### PR TITLE
MCS-1765:  Update Shape Constancy Weights for Eval 6

### PR DESCRIPTION
**Pull request by brianpippin**
You can see the weights make sense now as there are 500 unweighted, and 400 weighted scenes, as we are exluding 1/5 of the total cells.

Weights before the script:
Screenshot 2023-04-19 at 12 39 17 PM

Weights after the script:
Screenshot 2023-04-19 at 12 42 26 PM

### CODE REVIEW

File: scripts/_0_6_5_4_fix_shape_constancy_weight.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import mcs_history_ingest
+
+def correct_shape_constancy_weights(mongoDB):
+    results_collection = mongoDB["eval_6_results"]
+    scenes_collection = mongoDB["eval_6_scenes"]
+
+    # This will only get NYU agent tasks, "seeing leads to knowing" has
+    #   a test_type of "passive"
```
Comment (ThomasSchellenbergNextCentury) [inline scripts/_0_6_5_4_fix_shape_constancy_weight.py:8]: Not super important, but I don't think this comment is relevant here.

**Review by ThomasSchellenbergNextCentury — APPROVED** [review]

File: scripts/_0_6_5_4_fix_shape_constancy_weight.py (hunk 1 above)
Comment (brianpippin) [reply scripts/_0_6_5_4_fix_shape_constancy_weight.py:8]: I copied some of this from the other script, I will remove this.

**Review by phile-caci — APPROVED** [review]
tested locally, this worked for me.

before:
image

after:

image

**Comment by brianpippin** [comment]
I will merge this after the pair scoring script finishes running.

**Pull request merged**